src/LDNS.xs

```diff
@@ -1375,32 +1375,42 @@ packet_edns_version(obj,...)
 # -------------------
 # Get/set EDNS data
 # 
-# Beware, this code can only take a unique U32 parameter which means it 
-# is not a full implementation of EDNS data but it is enough for our 
+# This function will set EDNS RDATA in a packet when given a parameter,
+# otherwise it will return the entire EDNS RDATA of the packet (if any).
```

Comment:
Does "EDNS RDATA" mean RDATA of the OPT record? It is where EDNS option codes and option data are stored. I think it is better to refer to it as "OPT RDATA" or "EDNS option codes and data".

Reply:
> Does "EDNS RDATA" mean RDATA of the OPT record?

Yes.

> I think it is better to refer to it as "OPT RDATA" or "EDNS option codes and data".

Ok, I will rename them.